src/views/router_views/skeleton_one/skeleton_one.njk

```diff
@@ -9,8 +9,8 @@
   </h1>
 
       {{ govukButton({
-          text: "Skeleton Three",
-          href: Urls.SKELETON_THREE,
+          text: "Confirm and continue",
```

Comment:
you can get this text from `i18n.global_confirm_continue_button`

Reply:
Hopefully skeleton_one.njk is removed now